features/git-town-ship/current_branch/on_feature_branch/on_parent_branch.feature

```diff
@@ -42,11 +42,11 @@ Feature: git town-ship: shipping a parent branch
 
 
   Scenario: undo
-    When I run `git town-ship --undo`
+    When I run `gt ship --undo`
     Then it runs the commands
       | BRANCH         | COMMAND                                                      |
       | main           | git branch parent-feature <%= sha 'parent feature commit' %> |
-      |                | git revert <%= sha 'parent feature done' %>                  |
+      |                | git revert HEAD                                              |
```

Comment:
Tried doing the following and it gave a different sha:

* squash branch
* commit
* get sha
* reset to previous commit

The sha that resulted in ended up being different from the sha of the actual commit. I believe the sha includes the timestamp of the commit.